### ISSUE TITLE
Need help / documentation on bot commands

### ISSUE THREAD

**Issue by pizkaz** [silkeh/alertmanager_matrix#5]
First of all: Really cool project! I am currently evaluating this for our IT services. :-)

I am not an expert in Go - tbh I can barely read it - but if I understand the following lines correctly:

https://github.com/silkeh/alertmanager_matrix/blob/09e2578c57777db150323aa4c6531935cbe55a4a/pkg/bot/matrix.go#L59-L63
https://github.com/silkeh/alertmanager_matrix/blob/09e2578c57777db150323aa4c6531935cbe55a4a/pkg/bot/matrix.go#L77-L79

I should be able to type sth like `!alert list` and get a response from the bot, right?
Unfortunately this is not working for me. Am I missing something? Do I need to specify a command line option or set an environment variable for this feature?

**Comment by silkeh** [comment]
Yes, you understood correctly. You can send a command to the bot with `!alert`, `!alertmanager` and via highlight. The bot has a `help` command, so `!alert help` gives you a summary of all commands, and `!alert help list` the help for the `list` command and subcommands.

As long as the service connects and the configured user is present in the room it should work. Note that it currently doesn't accept invites automatically. The application should output something like this when it connects successfully:

```
2022/03/16 12:06:35 Connecting to Matrix homeserver at http://localhost:8008 as @<user>, and to Alertmanager at <alertmanager url>
2022/03/16 12:06:35 Listening on :4051
```

**Comment by pizkaz** [comment]
Hmm nope :-(

I am in a room with the bot and it sends nicely formatted alerting messages. So that part is working like a charm. But the bot ignores all messages from me.

Log excerpt:
```
Mar 16 02:29:54 alertmanager alertmanager_matrix[51162]: 2022/03/16 02:29:54 Connecting to Matrix homeserver at <redacted> as @<redacted>:<redacted>, and to Alertmanager at http://localhost:9093
Mar 16 02:29:54 alertmanager alertmanager_matrix[51162]: 2022/03/16 02:29:54 Listening on :4051
Mar 16 02:32:12 alertmanager alertmanager_matrix[51162]: 2022/03/16 02:32:12 Sending message to !<redacted>:<redacted>: 🚨 CRITICAL TestAlert: Foobarbaz (e8e93cdde5d6f3d1)
Mar 16 02:37:12 alertmanager alertmanager_matrix[51162]: 2022/03/16 02:37:12 Sending message to !<redacted>:<redacted>: ✅ RESOLVED TestAlert: Foobarbaz (e8e93cdde5d6f3d1)
```

**Comment by pizkaz** [comment]
Nevermind. Encryption was enabled in the room I was using.

Sorry for the noise.
